docs/getting-started/install-configure.md

```diff
@@ -1,20 +1,54 @@
 ---
-title: Install & Configure
-toc: true
-weight: 2
-indent: true
+title: Install & Configure toc: true weight: 2 indent: true
 ---
 
-# Install & Configure Crossplane
+# Choosing Hosted or Self-Hosted Crossplane
 
-Crossplane can be easily installed into any existing Kubernetes cluster using
-the regularly published Helm chart. The Helm chart contains all the custom
-resources and controllers needed to deploy and configure Crossplane.
+Users looking to use Crossplane for the first time have two options avaiable to
```

Comment:
```suggestion
Users looking to use Crossplane for the first time have two options available to
```

Reply:
Took Jared's advice and installed a spellchecker. :)